components/Timeline.tsx

```diff
@@ -0,0 +1,114 @@
+import Image from "next/image";
+import { motion } from "framer-motion";
+
+function Phase({ title, date, duration, imageSrc, phaseText }) {
+  return (
+    <motion.div
+      className="relative flex flex-row justify-between"
+      initial={{ opacity: 0 }}
+      whileInView={{ opacity: 1 }}
+      transition={{ duration: 1 }} // customize transition here
+      viewport={{ once: true }}
+    >
+      <div className="mr-4 flex w-10 flex-col items-center">
+        <div>
+          <div className="flex h-8 w-8 items-center justify-center rounded-full bg-blue-200 ">
+            <div className="h-4 w-4 rounded-full bg-blue-600"></div>
+          </div>
+        </div>
+        <div className="h-full w-px bg-blue-300"></div>
+      </div>
+      <div>
+        <h2 className="mb-4 inline-block rounded-3xl bg-gradient-to-r from-blue-500 to-blue-900 px-4 py-2 text-xs font-medium text-gray-100">
+          {date} <br />({duration})
+        </h2>
+        <div className="relative mb-10 flex flex-1 flex-col rounded-3xl border-b-4 border-blue-200 bg-white shadow">
+          <Image
+            src={imageSrc}
+            alt={title}
+            width={0}
+            height={0}
+            sizes="100vw"
+            style={{ width: "100%", height: "auto" }} // optional
+          />
+
+          <div className="z-20 p-6">
+            <p className="mb-2 text-xl font-bold text-gray-900">{title}</p>
+            <p className="text-gray-800">{phaseText}</p>
+          </div>
+        </div>
+      </div>
+    </motion.div>
+  );
+}
+
+export default function Timeline() {
```

Comment:
We try to keep everything in components/ reusable. In this case we this component should have the Props

```ts
interface TimelineProps {
phases: {
  title: string,
  date: string,
  duration: string,
  image: string,
  text: string,
  }[]
}
```

The data that is used should be on the page